data/mods/Xedra_Evolved/snippets/e_logs.json

```diff
@@ -0,0 +1,46 @@
+      },
+      {
+        "id": "local_files_xedra_4",
+        "text": "*cached messages in some chat app* \nB0yscout(Bravo 7): so, do you guys know what's going on?\nJackTrader(PD): wdym?\nB0yscout(Bravo 7): i mean the fuck is going on with the world? when i just came here, you guys tried to solve a vampire disease, then a shifter overpopulation, and yesterday my unit happened to fight A MOTHERFUCKING ANGEL!  ANGEL!!!\nB0yscout(Bravo 7): am i tripping, or are we living in some junk novel, written by idiots without any sense of… eeh… you get it\nJackTrader(PD): oh\nJackTrader(PD): …\nJackTrader(PD): hmm, what do i say\nJackTrader(PD): it's actually pretty serious topic\nJackTrader(PD): i'd even say it's impossible to solve, because we can't really measure why happens this, and why happens that\nJackTrader(PD): but\nB0yscout(Bravo 7): :eyes:\nJackTrader(PD): we talked about it with @WHO and a few other folks from the paranormal department, and we think there are a few possibilities\nJackTrader(PD): the official one is that the portal shit is so massive, that the probability of us meeting something, that resembles our folklore is not that small\nJackTrader(PD): like you met a wheeled ball with feathers yesterday, but did you saw a square of light, that speaks perfect english?\nJackTrader(PD): i did\nB0yscout(Bravo 7): i'm not surprised\nB0yscout(Bravo 7): i guess the juice is in the unofficial versions?\nJackTrader(PD): the first unofficial version, that we hope is correct, is that there were portal outbreaks before, and it's just not the first time planet Earth fights the vampire invasion or shit\nB0yscout(Bravo 7): why isn't it official?\nJackTrader(PD): because people are idiots, mostly.   Imagine if POTUS made a speech saying \"do you remember all this shit about satan and unicorns and loch ness?  well it's actually true.   except satan.  maybe\"\nB0yscout(Bravo 7): and?\nJackTrader(PD): you'd be surprised how much overlap there is between people who ~~think~~  thought unicorns are real, and people who think we should kill all jews\nB0yscout(Bravo 7): okay that makes sense\nB0yscout(Bravo 7): that's the only unofficial theory you discuss?\nJackTrader(PD): we thought about another one, but it's much, much worse\nJackTrader(PD): that the stuff, that invades our world, can connect to some sort of noosphere and thing, and can read our thoughts\nJackTrader(PD): as they know everything we are afraid of\nJackTrader(PD): demons\nJackTrader(PD): vampires\nJackTrader(PD): diseases\nJackTrader(PD): nuclear war\nB0yscout(Bravo 7): …\nB0yscout(Bravo 7): …\nB0yscout(Bravo 7): is it classified?\nJackTrader(PD): …oh shit "
```

Comment:
```suggestion
        "text": "*cached messages in some chat app* \nB0yscout(Bravo 7): So, do you guys know what's going on?\nJackTrader(PD): wdym?\nB0yscout(Bravo 7): I mean the fuck is going on with the world?  When I just came here, you guys tried to solve a vampire disease, then a shifter overpopulation, and yesterday my unit happened to fight A MOTHERFUCKING ANGEL!  ANGEL!!!\nB0yscout(Bravo 7): Am I tripping, or are we living in some junk novel, written by idiots without any sense of… eeh… you get it.\nJackTrader(PD): oh\nJackTrader(PD): …\nJackTrader(PD): Hmm, what do I say?\nJackTrader(PD): It's actually a pretty serious topic\nJackTrader(PD): I'd even say it's impossible to solve, because we can't really measure why happens this, and why happens that.\nJackTrader(PD): but\nB0yscout(Bravo 7): :eyes:\nJackTrader(PD): We talked about it with @WHO and a few other folks from the paranormal department, and we think there are a few possibilities.\nJackTrader(PD): The official one is that the portal shit is so massive, that the probability of us meeting something, that resembles our folklore is not that small.\nJackTrader(PD): Like you met a wheeled ball with feathers yesterday, but did you see a square of light, that speaks perfect English?\nJackTrader(PD): I did\nB0yscout(Bravo 7): I'm not surprised\nB0yscout(Bravo 7): I guess the juice is in the unofficial versions?\nJackTrader(PD): The first unofficial version, that we hope is correct, is that there were portal outbreaks before, and it's not the first time planet Earth fights the vampire invasion or shit.\nB0yscout(Bravo 7): Why isn't it official?\nJackTrader(PD): Because people are idiots, mostly.   Imagine if POTUS made a speech saying \"Do you remember all this shit about Satan and unicorns and loch ness?  Well it's actually true.  Except Satan.  Maybe\"\nB0yscout(Bravo 7): and?\nJackTrader(PD): You'd be surprised how much overlap there is between people who ~~think~~  thought unicorns are real, and people who think we should kill all Jews.\nB0yscout(Bravo 7): Okay that makes sense\nB0yscout(Bravo 7): That's the only unofficial theory you discuss?\nJackTrader(PD): We thought about another one, but it's much, much worse.\nJackTrader(PD): That the stuff, that invades our world, can connect to some sort of noosphere and thing, and can read our thoughts,\nJackTrader(PD): as they know everything we are afraid of:\nJackTrader(PD): demons\nJackTrader(PD): vampires\nJackTrader(PD): diseases\nJackTrader(PD): nuclear war.\nB0yscout(Bravo 7): …\nB0yscout(Bravo 7): …\nB0yscout(Bravo 7): Is it classified?\nJackTrader(PD): …oh shit."
```
~~Does this even count as typo fixing when these are in-universe cached app messages where people don't practice perfect diction~~